frontend/app/services/letters-service.server.ts

```diff
@@ -105,6 +118,7 @@ function createLettersService() {
     });
 
     if (!response.ok) {
+      instrumentationService.countHttpStatus('http.client.interop-api.letters.posts', response.status);
```

Comment:
I you move this line of code outside of the `if (..)`, then you would only need to call `.countHttpStatus(..)` once in this function.

This comment applies to other similar code in this PR.

Reply:
Would I just use the one `instrumentationService.countHttpStatus('http.client.interop-api.letters.posts', response.status)` then? (and not the `instrumentationService.countHttpStatus('http.client.interop-api.letters.posts', 200);` outside it)

Reply:
Yes.. that is correct.

Reply:
Fixed: b54fdc1071cc2f7076d56ba903332d1f0b525eed